okhttp/src/test/java/okhttp3/KotlinSourceCompatibilityTest.kt

```diff
@@ -891,6 +891,7 @@ class KotlinSourceCompatibilityTest {
     builder = builder.eventListenerFactory(object : EventListener.Factory {
       override fun create(call: Call): EventListener = TODO()
     })
+    builder = builder.eventListenerFactory(LoggingEventListener.Factory { s -> TODO() })
```

Comment:
This is a source test of LoggingEventListener.Factory, but included here to show working end to end.